### PR TITLE
About Session

### PR DESCRIPTION
**Pull request by MeSukani**
Hey guys... I need your help with the session, I'm stuck somewhere. as Damian wrote in the auth.php file that we have to take values from the post method from landing.php and push it to the log table, but inside the log table there isn't any field for email or first name, or last name where can I connect with the session I guess so I changed with user table and now I don't know where is the problem create could you correct me where can problem appears...

basically, I did something like this, so if the user can log in with their account it will store data in session and it will be transferred to index.php and if the user cannot login user will stay on landing.php

### CODE REVIEW

**Comment by damianulan** [comment]
When u JOIN log table and user table using FK on user_id field, there will nothing be missing. It's all there in database.

**Comment by damianulan** [comment]
@MeSukani Please be on tomorrow's meeting at 6pm.

**Review by FabulouslyGuy — APPROVED** [review]

**Review by FabulouslyGuy — APPROVED** [review]

**Review by damianulan — APPROVED** [review]
